mesheryctl/pkg/utils/helpers.go

```diff
@@ -341,12 +358,36 @@ func installprereq() error {
 }
 
 // IsMesheryRunning checks if the meshery server containers are up and running
-func IsMesheryRunning() bool {
-	op, err := exec.Command("docker-compose", "-f", DockerComposeFile, "ps").Output()
-	if err != nil {
-		return false
+func IsMesheryRunning(currPlatform string) bool {
+	switch currPlatform {
+	case "docker":
+		{
+			op, err := exec.Command("docker-compose", "-f", DockerComposeFile, "ps").Output()
+			if err != nil {
+				return false
+			}
+			return strings.Contains(string(op), "meshery")
+		}
+	case "kubernetes":
+		{
+			client, err := meshkitkube.New([]byte(""))
+
+			if err != nil {
+				return false
+			}
+
```

Comment:
`err != nil` would mean that it fails to create the client. So ideally, it should be returning the error. Otherwise, it would be wrong error stating that Meshery is not running...